src/lib/components/blog/blog-entry.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { makeBlogPostPageUrl } from '$lib/public/blog/urls';
+  import { readingTime } from '$lib/public/blog/utils';
+  import { Temporal } from '@js-temporal/polyfill';
+  import Labels from '../../../routes/blog/labels.svelte';
+
+  export let post: any;
```

Comment:
The BlogPost type definition is exported from `src/lib/public/blog/types.ts` and should be used as the type for the variable `post`.

Reply:
addressed